workflow/Snakefile

```diff
@@ -137,4 +140,55 @@ rule sniffles:
 			mem_mb= 32000,
 			runtime= 1440   
 	shell:
-			"sniffles --input {input.bam} --vcf {output}"
\ No newline at end of file
+			"sniffles --input {input.bam} --vcf {output}"
+
+rule cutesv:
+	input:
+			bam="results/sorted/{sample}.sorted.bam",
+			gen="resources/GRCh38.primary_assembly.genome.fa",
+	output:
+			"results/cutesv/{sample}_cutesv.vcf"
+	conda:
+			"envs/cutesv.yaml"
+	benchmark: "benchmarks/{sample}.tsv"
+	threads: 
+			8
+	resources: 
+			mem_mb= 32000,
+			runtime= 1440   
+	shell:
+        	"cuteSV {input.bam} {input.gen} {output} results/"
+
+# rule nanomonsv_parse:
+# 	input:
+# 			bam="results/sorted/{sample}.sorted.bam",
+# 			sniffles="results/sniffles/{sample}.vcf"
+# 	output:
+# 			directory("results/nanomonsv_parse/{sample}")
+# 	conda:
+# 			"envs/nanomonsv.yaml"
+# 	benchmark: "benchmarks/{sample}.tsv"
+# 	threads: 
+# 			8
+# 	resources: 
+# 			mem_mb= 32000,
+# 			runtime= 1440   
+# 	shell:
+# 			"nanomonsv parse {input.bam} {sample}"
+
+# rule nanomonsv_get:
+# 	input:
+# 			"results/nanomonsv_parse/{sample}"
+# 	output:
+# 			"results/nanomonsv_get/{sample}.vcf"
+# 	conda:
+# 			"envs/nanomonsv.yaml"
+# 	benchmark: "benchmarks/{sample}.tsv"
+# 	threads: 
+# 			8
+# 	resources: 
+# 			mem_mb= 32000,
+# 			runtime= 1440   
+# 	shell:
+# 			"nanomonsv parse {input.bam} {sample}"
+
```

Comment:
You should keep this in another branch if you like, but commented code should ideally not make it to the main branch